cypress/integration/pages/onDemandRadio/testsForCanonicalOnly.js

```diff
@@ -12,5 +14,47 @@ export default ({ service, pageType }) => {
         });
       }
     });
+    describe('Radio Schedule', () => {
+      it('should be displayed if there is enough schedule data', () => {
+        const toggleKey = 'onOnDemandRadioPage';
+        const isRadioScheduleOnPage = serviceHasRadioSchedule({
+          service,
+          variant,
+          toggleKey,
+        });
+        cy.log(
+          `On Demand Radio Page configured for Radio Schedule? ${isRadioScheduleOnPage}`,
+        );
+
+        if (isRadioScheduleOnPage) {
+          const schedulePath = Cypress.env('currentPath');
```

Comment:
I created a utility to get the schedule path which you could use - see https://github.com/bbc/simorgh/blob/6e71e408411cd199d8258fe1510bdd4c4b2fe1ad/cypress/integration/pages/liveRadio/helper.js#L23 (please note it's on a different branch, and does not appear to have been merged to latest)

Reply:
Ignore this - it only works for live radio